spiketoolkit/validation/quality_metrics.py

```diff
@@ -22,6 +22,7 @@
 
 def compute_num_spikes(
         sorting,
+        duration,
```

Comment:
This is not used. I think MetricData could allow duration=None (like in this case)

Reply:
Good point!

Reply:
Maybe we can also allow duration to be `None`. If not given max(spiketimes) - min(spiketimes) will be used. What do you think?

Reply:
Oh, I thought we wanted duration to be mandatory. If we allow for it to be optional, then the value will change if you pass a RecordingExtractor in or not.

Reply:
maybe `duration` suggests the duration in seconds. `duration_in_frames`? 
I think we could also add `get_duration()` in the `RecordingExtractor` (maybe in the `SortingExtractor` too, similarly to sampling_frequency.

Reply:
I can call it `num_frames`. We already have `get_num_frames` in the RX which is equivalent since all of our extractors start at frame 0.

Reply:
Called it `duration_in_frames` as you mentioned.